public/locales/en/translation.json

```diff
@@ -323,6 +323,13 @@
     },
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
+    },
+    "contactUsInfo": {
+      "header": "Leave us a message",
+      "description": "Safe place is here for you. Please use this form to contact us, share your feedback, submit a request for assistance or work toghther to end gender-based violence.",
```

Comment:
Safe place or Safe Place? i think the latter

Reply:
typo in together

Reply:
done